README.md

```diff
@@ -94,17 +94,23 @@ To run tests locally:
 To add tests:
 
 - Tests must be in files beginning with `test-`, and be in the `tests/cromwellapi/` dir
-- Ideally use the pytest fixture `cromwell_api` created in `tests/cromwellapi/conftest.py` to call the Cromwell API. If there's a method missing that you need add it to the `CromwellApi` class in `tests/cromwellapi/cromwell.py`
+- Ideally use the pytest fixtures `cromwell_api` or `cromwell_api_final` - created in `tests/cromwellapi/conftest.py` - to call the Cromwell API. If there's a method missing that you need add it to the `CromwellApi` class in `tests/cromwellapi/cromwell.py` or `CromwellApiFinal` class in `tests/cromwellapi/cromwell_final.py`. See existing test files for patterns.
+- Run tests by limiting the WDL folders used to just those of interest for your new test code. This is because we only want to generate new "cassettes" associated with tests associated with the changes being made - instead of for all tests. To do so, don't use the make commands as above. Instead use something like `op run -- uv run pytest -n auto --color=yes --record-mode=rewrite --verbose -s tests/cromwellapi/ -k wdlFolderName`; replacing `wdlFolderName` with the name of the WDL folder you want to test. The `-k` flag is used to filter tests based on their names or attributes. Once you've recorded new cassettes, run the `pytest` line again but with `--record-mode=once`.
```

Comment:
Seeing if I understand: `--record-mode=once` means that when you run the test it will use the existing cassette, which will typically be a cassette you recently recorded.

Reply:
Yes, thanks.